### ISSUE TITLE
Nintendo Switch builds for 1.12.3 release

### ISSUE THREAD

**Issue by asdone7** [hrydgard/ppsspp#15252]
### What should happen

I noticed that1.12.3 was published, however, there are no Nintendo Switch binaries despite

Can you release a binary file of Nintendo switch version?

Thank you in advance

### Who would this benefit

Nintendo switch user

### Platform (if relevant)

Switch

### Games this would be useful in

none

### Checklist

- [X] Check the [latest git build](https://buildbot.orphis.net/ppsspp/) in case it's already implemented.
- [X] [Search for other requests](https://github.com/hrydgard/ppsspp/search?q=my+issue&type=issues) of the same feature.

**Comment by unknownbrackets** [comment]
Unfortunately, the Switch version is still being maintained somewhat separately and needs some work to update.  See #12323.

I don't think there's been an update for a little while:
https://www.patreon.com/posts/28559579

-[Unknown]

**Comment by m4xw** [comment]
Just saw this... Yea i failed to get the required time lately. Hoped to be able to rebase by now but... sigh.
fyi i rarely post on my patreon for updates. It's all on discord so it doesnt represent when i work on stuff.
Last update was dated Sep 27, 2021 but it never made the bump to upstream release page because the GLES2 variants are broken since mid last year for my hackery

**Comment by m4xw** [comment]
@asdone7 https://m4xw.net/nextcloud/index.php/s/LHdrXozpfFMT28M

**Comment by Bassnaut93** [comment]
@m4xw is this supposed to 1.12.3? Cause the download is for some branch of 1.11.3.

**Comment by unknownbrackets** [comment]
v1.11.3-2764 actually means "v1.11.3 plus 2764 changes" (there's a hash after that which identifies which specific changes, because you can imagine two people might both change v1.11.3 different ways, so a count doesn't tell you everything.)

The most recent git build is v1.12.3-1158-g287dd66f2.  But you could also call it v1.11.3-2898-g287dd66f2.

This just means that when the latest Switch build was created, tags weren't fetched.  So their git didn't know about v1.12.3, and just labeled the build based on changes since v1.11.3.  As you can work out from the above numbers, v1.12.3 has about 1,740 changes in it compared to v1.11.3.

In other words, the version you're using is actually newer than v1.12.3.

-[Unknown]

**Comment by asdone7** [comment]
@m4xw Just saw this , thank you very much. I have received your release。

**Comment by amarantsnipersnake** [comment]
> v1.12.3-1158-g287dd66f2

im just wondering where is PPSSPP_GL2.nro file?
